packages/e2e/test/web-extension/extension/background/walletManager.ts

```diff
@@ -25,12 +25,19 @@ import { runtime } from 'webextension-polyfill';
 
 const walletFactory: WalletFactory = {
   create: async (
-    { observableWalletName, cardanoServicesUrl: _cardanoServicesUrl }: WalletManagerActivateProps,
+    props: WalletManagerActivateProps,
     dependencies: { keyAgent: AsyncKeyAgent; stores: storage.WalletStores }
   ) =>
-    // cardanoServicesUrl could be used to construct providers for different networks
+    /* provider could be used to pass the necessary information to construct providers for different networks.
```

Comment:
Is this duplication necessary? The type has the documentation, so it's not clear to me why we'd want it here too

Reply:
It should not be duplicated. I have updated it in here: [9f63efb](https://github.com/input-output-hk/cardano-js-sdk/pull/502/commits/9f63efb609b9314b08cce9580ec96f9108303122)